src/patternfly/components/Select/select-chip-group-collapsed.hbs

```diff
@@ -0,0 +1,28 @@
+{{#> chip}}
+  {{#> chip-text chip-text--attribute=(concat 'id="' id '-chip_one"')}}
+    Arkansas
+  {{/chip-text}}
+  {{#> button button--modifier="pf-m-plain" button--HasTimesCircleIcon="true" button--attribute=(concat 'aria-labelledby="remove_' id '_chip_one ' id '-chip_one" aria-label="Remove" id="remove_' id '_chip_one"')}}
```

Comment:
```suggestion
  {{#> button button--modifier="pf-m-plain" button--attribute=(concat 'aria-labelledby="remove_' id '_chip_one ' id '-chip_one" aria-label="Remove" id="remove_' id '_chip_one"')}}
    {{> button-icon button-icon--type="times-circle"}}
```

Reply:
These button icon updates are a result of https://github.com/patternfly/patternfly-next/pull/1141